readthedocs/rtd_tests/tests/test_doc_serving.py

```diff
@@ -260,3 +274,11 @@ def test_sitemap_xml(self):
                 private=True,
             ),
         )
+        self.assertNotContains(
```

Comment:
This test looks tricky to me.

We are adding a new public translated version of `stable`, which I expect to be listed in the response, but we are asserting to not contains. Why it does not appear in the response?

Reply:
I think the test we want to is a project with 3 public versions and only 1 version translated. So, in the response 2 should not appear and 1 should.

I'm not sure if I'm confused or missing something here.

Reply:
@humitos Here the public project has 2 public versions `latest` and `stable` (ie: Latest version is created automatically). but the translation project has only one version which is `latest`. so, I'm checking that as the translation project does not have a `stable` version, `stable` should not have an URL for that Translation in the sitemap.

Reply:
Makes sense.

Can you add this explanation as a comment above the assert line?

When creating this tests, I like to use descriptive names on things when possible. For example, the slug for the `stable` version in this case could be `not-translated-version` (`stable` is confusing since it's also created automatically).

Besides, the slug for the translated project could be something similar to `translation-es` and similar.

Reply:
Then, when checking for the assert everything is clearer:

```
          self.public.get_docs_url(
                version_slug=not_translated_version.slug,
                lang_slug=translation.language,
                private=False,
            ),
```

It easier to note that the "not translated version" should not be shown on the sitemap xml because it's not translated :)

Reply:
@humitos Thanks for Pointing it out. I'll keep that in mind. I have updated the PR have a Look.